.gitignore

```diff
@@ -0,0 +1,2 @@
+books.js
+recipes.js

```

Comment:
why are these 2 files in your .gitignore? If they are not needed go ahead and delete them 🧹